src/utils/helpers.js

```diff
@@ -21,7 +21,12 @@ export const groupBy = (arr, key) => {
   );
 };
 
-export const isMobile = _ =>
-  /Android|webOS|iPhone|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
-    navigator.userAgent
-  );
+export const isMobile = _ => {
+  if (typeof navigator !== "undefined") {
+    return /Android|webOS|iPhone|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
+      navigator?.userAgent
```

Comment:
A suggestion that you don't have to take!

Seems like you could instead do:

```
{
  if (!navigator?.userAgent) return false; // for build step
  
  return /Android|webOS|iPhone|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
      navigator.userAgent 
  )
}
```

Background:

Sniffing for specific features in an API to test browser support (or lack thereof) is a normal way to include a polyfill (or progressive enhancement) or not

Reply:
I tried this exact syntax, but it was throwing an error that `navigator is not defined` Don't totally understand why as I thought `?.` would work.